android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeModule.kt

```diff
@@ -78,8 +77,7 @@ class CustomerIOReactNativeModule(
                 region = region,
                 appContext = reactApplicationContext.applicationContext as Application,
             ).apply {
-                // TODO: Update when sdk version is released
-                setClient(Client.ReactNative)
+                setClient(Client.ReactNative(sdkVersion = sdkVersion ?: "n/a"))
```

Comment:
Is `n/a` the default value for sdkVersion?

Reply:
No, not really. This will be used only if nothing is provided from the React app. I added `n/a` instead of empty string so the case of not providing value can be differentiated.

Reply:
Make sense. But I doubt if that will ever happen i.e. the package not providing the version.

Reply:
Yes, this should never happen ideally. So this is just a safe fallback that should ideally never appear 👀